app/src/main/res/values-es/strings.xml

```diff
@@ -6,6 +6,8 @@
     <string name="cancel">Cancelar</string>
     <string name="ok">Aceptar</string>
     <string name="unknown_error">¡Vaya! Algo salió mal.</string>
+    <string name="copy">Copy</string>
```

Comment:
Please move strings related to copy, such as button title, toast message, and content description, to the [goal info screen block](https://github.com/Pool-Of-Tears/GreenStash/blob/823c0e1bf183e2e3809cf17e7b056cfb87b8cca5/app/src/main/res/values/strings.xml#L81) inside `strings.xml`. Additionally, as a suggestion, if possible, you can also add translations for these strings. As they're very small (ranging from one to a few words), it shouldn't be a problem. You can use ChatGPT's help for translation.

Reply:
Strings moved and renamed to have a matching prefix in [42ed507](https://github.com/Pool-Of-Tears/GreenStash/pull/95/commits/42ed50700bd89d421b0cdc8cda02203326502405)

As for translations, I will look into them slightly later.

Reply:
Thanks! Translation was just a suggestion and if you want, you can also skip it, I'll add them later :)

Reply:
Then lets leave the translations as is for now. Can work on them more with later PRs.